.ci/scripts/test.sh

```diff
@@ -144,8 +145,9 @@ if [[ $BUILD_TYPE != "release" && $FORCE != "true" ]]; then
   #
   # Note: We are relying on new releases being added to the top of index.tab,
   # which currently seems to be the case.
-  latest_edge_version=$(curl -sS ${NVM_NODEJS_ORG_MIRROR}/index.tab \
-    | (grep "^v${NODE_VERSION}" || true) | awk '{print $1}' | head -1)
+  index_tab_content=$(curl -sS "${NVM_NODEJS_ORG_MIRROR}/index.tab" \
+    | (grep "^v${NODE_VERSION}" || true) | awk '{print $1}')
+  latest_edge_version=$(echo "$index_tab_content" | head -1)
   if [[ -z "$latest_edge_version" ]]; then
```

Comment:
I see here we've broken up the previous one liner into two lines.  I also am not sure why this second version works more consistently and the first version does not, but this does not appear to do any harm so 👍